src/ucp/core/ucp_request.h

```diff
@@ -58,6 +58,8 @@ typedef struct ucp_frag_state {
         struct {
             size_t                iov_offset;     /* Offset in the IOV item */
             size_t                iovcnt_offset;  /* The IOV item to start copy */
+            size_t                iovcnt;         /* Number of IOV buffers */
+            uct_mem_h             memh;           /* Pointer to IOV memh[iovcnt] */
```

Comment:
`sizeof(ucp_request_t)`: 184->200 bytes
Proposal is to move `iovcnt` into datatype as it used in `contig` data type.